demo/src/screens/componentScreens/BadgesScreen.js

```diff
@@ -1,25 +1,68 @@
 import React, {Component} from 'react';
 import {ScrollView, StyleSheet} from 'react-native';
-import {View, Badge, Colors} from 'react-native-ui-lib';//eslint-disable-line
+import {Colors, View, Badge, Button, Text} from 'react-native-ui-lib';//eslint-disable-line
 
 const BadgesSpace = 30;
 
 export default class BadgesScreen extends Component {
 
+  constructor(props) {
```

Comment:
you can avoid adding a constructor if you just need to initialize the state. 
you can just add a state member in the Class, it's cleaner.. 

```
Class BadgeScreen extends Component {
  state = {
    value: 1
  }
  ....
}
```